src/BSP/x_nucleo_nfc04a1_nfctag.cpp

```diff
@@ -0,0 +1,297 @@
+/**
+  ******************************************************************************
+  * @file    x_nucleo_nfc04a1_nfctag.c
```

Comment:
This file should be renamed to `nfctag.c` or `st25dv_nfctag.c` and all `x_nucleo_nfc04a1_` have to be removed.
```suggestion
  * @file    st25dv_nfctag.c
```

Reply:
Done